src/actions/authAction.js

```diff
@@ -0,0 +1,75 @@
+import {
+  REGISTER_SUCCESS,
+  REGISTER_FAIL,
+  USER_LOADED,
+  AUTH_ERROR,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+  LOGOUT,
+  CLEAR_ERRORS
+} from './types';
+import Requestor from '../components/Requestor';
+import setAuthToken from '../utils/setAuthToken';
+
+const config = {
+  'Content-Type': 'application/json'
+};
+
+export const register = user => async dispatch => {
+  try {
+    const response = await Requestor.post('/users', config, user);
+
+    window.location = '/confirmation-send';
+
+    dispatch({
+      type: REGISTER_SUCCESS,
+      payload: response.data
+    });
+  } catch (error) {
+    dispatch({
+      type: REGISTER_FAIL,
+      payload: error.response.data.msg
+    });
+  }
+};
+
+export const login = user => async dispatch => {
+  try {
+    const response = await Requestor.post('/auth', config, user);
+
+    window.location = '/';
```

Comment:
Same comment here, should reroute user at the component level instead of actions level